### ISSUE TITLE
Integer variables

### ISSUE THREAD

**Issue by mbanco** [jtilly/knitroR#3]
Hi,
Excellent work!!
Is there any option to define integer or binary variables?

Best Regards,

**Comment by jtilly** [comment]
No, not in the current package. Conceptually, this shouldn't be very difficult. You would need to adjust the wrapper in [R/knitro.R](https://github.com/jtilly/knitroR/blob/master/R/knitro.R) and then modify the function `knitroCpp()` in [src/knitro.cpp](https://github.com/jtilly/knitroR/blob/master/src/knitro.cpp).
